projects/batfish-client/src/main/java/org/batfish/client/QuestionHelper.java

```diff
@@ -41,4 +43,35 @@ public static String getQuestionString(
       return question.toJsonString();
     }
   }
+
+  /**
+   * Merges local and remote questions and overwrites remote with local
+   *
+   * @param localQuestions Questions from disk
+   * @param remoteQuestions Questions from Coordinator
+   * @param bfQuestions Batfish Questions
+   * @param ae Answer Element
+   * @return Merged {@link Map}
+   */
+  public static LoadQuestionAnswerElement mergeQuestions(
```

Comment:
Why is this function here and public if it's only used in the other file?

Put it there, and make it private. (or default-visibility for testing).

Reply:
moved to Client class